cluster/addons/addon-manager/Makefile

```diff
@@ -15,7 +15,7 @@
 IMAGE=gcr.io/k8s-staging-addon-manager/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v9.1.5
+VERSION=v9.1.6
```

Comment:
when you open the PR to bump to this image version, also update the changelog to note the fix